src/metrics/chaoss.ts

```diff
@@ -295,3 +295,44 @@ FORMAT JSONCompact`;
     }
   });
 };
+
+export const chaossChangeRequestsAcceptedRatio = async (config: QueryConfig) => {
+  config = getMergedConfig(config);
+  const whereClauses: string[] = ["type = 'PullRequestEvent' AND action = 'closed'"];
+  const repoWhereClause = getRepoWhereClauseForClickhouse(config);
+  if (repoWhereClause) whereClauses.push(repoWhereClause);
+  whereClauses.push(getTimeRangeWhereClauseForClickhouse(config));
+
+  const sql = `
+SELECT
+  id,
+  argMax(name, time) AS name,
+  ${getGroupArrayInsertAtClauseForClickhouse(config, { key: 'accepted'})}
```

Comment:
You should add a comma at the tail of the line.

Reply:
Thanks！